.rubocop.yml

```diff
@@ -51,4 +51,6 @@ Style/GlobalVars:
     - "lib/webmachine/decision/conneg.rb"
 
 AllCops:
+  NewCops: disable
   SuggestExtensions: false
+  TargetRubyVersion: 2.6
```

Comment:
Attempt to make the CI build more stable by limiting changes in future Rubocop releases:

- Don't run new cops: this avoids failures when a new cop is introduced that disagrees with the current style.
- Target the minimum version of Ruby supported by the webmachine. This avoids Rubocop suggestions to use features only available in later versions of Ruby.